src/services/auth.service.js

```diff
@@ -0,0 +1,35 @@
+import nodemailer from 'nodemailer';
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  secure: true,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+  tls: {
+    servername: 'smtp.gmail.com',
```

Comment:
The `servername` in the `tls` configuration is hardcoded to 'smtp.gmail.com'. If you are using a different SMTP provider, make sure to update this value accordingly.